src/test/java/pages/DominoPage.java

```diff
@@ -0,0 +1,88 @@
+package pages;
+
+import elements.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.ConfigStorage;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class DominoPage extends BasePage {
+
+    private final String CATALOGUE_URL = ConfigStorage.getCatalogueUrl();
```

Comment:
Also why do you need CATALOGUE_URL in each PO? isn't that part of CATALOGUE_Page only?

Reply:
![image](https://user-images.githubusercontent.com/71931457/145398912-4a897602-e110-4a74-898c-d81fe627f397.png)
Yes, we need it almost in each PO
![image](https://user-images.githubusercontent.com/71931457/145398996-de6abddf-a091-4a7b-980c-81033c4570ec.png)